content/en/docs/how-tos/cluster-claim.md

```diff
@@ -183,6 +183,30 @@ stringData:
 type: Opaque
 ```
 
+### Set up RBACs for debugging
```

Comment:
Let's add this as a second step, after creating the namespace, so that once-per-namespace actions are together.